### PR TITLE
Add Hindi Translation for main page

### PR DESCRIPTION
**Pull request by MetalDent**
Translated few pages into Hindi.

Hope the format and spellings are correct.

### CODE REVIEW

File: _data/strings.yml [hunk 1]
```diff
@@ -19,3 +19,9 @@ locales:
     language_name: "中文"
     language: "官话"
     qmk_firmware: "QMK固件"
+
```
Comment (parammittal16) [inline _data/strings.yml]: I think space is necessary here?

**Review by parammittal16 — COMMENTED** [review]
LGTM :) Only that space one issue.

File: _data/strings.yml [hunk 2]
```diff
@@ -19,3 +19,9 @@ locales:
...
+  hi: &DEFAULT_HI
+    <<: *DEFAULT_EN
+    language_name: "हिंदी"
+    language: "भाषा"
+    qmk_firmware: "QMK-फर्मवेयर"
```
Comment (noroadsleft) [inline _data/strings.yml:60]: Key-value pairs for `made_possible`, `hosted_on`, and `description` are also needed here.

See below for the English text.

https://github.com/qmk/qmk.fm/blob/27bc9ef1e3de79a21d387d4298201fb985dc3b4f/_data/strings.yml#L2-L8

File: _i18n/hi/changes.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+title: "QMK में सभी हाल के बदलाव"
+layout: qmk-title
+permalink: /hi/changes/
+---
```
Comment (noroadsleft) [inline _i18n/hi/changes.md:5]:
```suggestion
lang: "hi"
---
```

File: _i18n/hi/changes.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+{% for change in site.changes %}
```
Comment (noroadsleft) [inline _i18n/hi/changes.md:7]: A recent change to upstream.

```suggestion
{% for change in site.changes reversed %}
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

File: _i18n/hi/sidebar.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+[![Build Status](https://travis-ci.org/qmk/qmk_firmware.svg?branch=master)](https://travis-ci.org/qmk/qmk_firmware)
+[![Discord](https://img.shields.io/discord/440868230475677696.svg)](https://discord.gg/Uq7gcHh)
+[![Docs Status](https://img.shields.io/badge/docs-ready-orange.svg)](https://docs.qmk.fm)
+[![GitHub contributors](https://img.shields.io/github/contributors/qmk/qmk_firmware.svg)](https://github.com/qmk/qmk_firmware/pulse/monthly)
+[![GitHub forks](https://img.shields.io/github/forks/qmk/qmk_firmware.svg?style=social&label=Fork)](https://github.com/qmk/qmk_firmware/)
+
+### पेज
+
+* [ऑनलाइन  कॉन्फ़िगरातोर ](https://config.qmk.fm)
+* [QMK में हाल के बदलाव](/changes/)
```
Comment (noroadsleft) [inline _i18n/hi/sidebar.md:18]:
```suggestion
* [QMK में हाल के बदलाव](/hi/changes/)
```

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**